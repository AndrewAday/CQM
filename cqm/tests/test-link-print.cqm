int main() {
  printf("hello world\r\n");
  printf("%f I am a float %d\r\n", 3.14, 29);
  printb(true);
  print(56);
}
