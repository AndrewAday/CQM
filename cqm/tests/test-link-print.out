hello world
3.140000 I am a float 29
1
56
